src/google-contetnt-provider.ts

```diff
@@ -0,0 +1,232 @@
+import config from '../config.json';
+import * as Storage from '@google-cloud/storage'
+
+const serviceAccount = config.providerOptions.serviceAccount;
+const bucketName = config.providerOptions.bucketName;
+const fileName = config.providerOptions.filename;
+const baseUrl = config.providerOptions.baseUrl;
+const filePathLocal =config.providerOptions.filePathLocal;
+
+  /**
+   * Check validity of Service Account configuration
+   * @param config
+   * @returns {{private_key}|{client_email}|{project_id}|any}
+   */
+
+  export function checkServiceAccount (config :any) {
```

Comment:
This utility methods can be moved to a separate file so that this file will only contain the `GoogleContentProvider` class and its methods.